Sources/ComposableCoreMotion/Internal/Exports.swift

```diff
@@ -1,2 +1,4 @@
+#if canImport(CoreMotion)
```

Comment:
these are necessary to get tvOS swiftui previews building. they for some reason try building ComposableCoreMotion, even though it isn't linked, and that fails since tvOS doesnt have core motion.